tests/.secrets.env.example

```diff
@@ -1,5 +1,3 @@
-export ZENKO_STORAGE_ACCOUNT_ACCESS_KEY=
```

Comment:
Is this an intended change?

Reply:
No.  My suspicion is that this is an unfortunate byproduct of my Git legerdemain, but I'm not good enough at it to even know when I've fscked it up.

Reply:
I copied the file over from the dev/0.9 branch. I hope that's the end of this issue.